src/k8s/pkg/component/loadbalancer.go

```diff
@@ -72,7 +73,12 @@ func UpdateLoadBalancerComponent(ctx context.Context, s snap.Snap, isRefresh boo
 	formattedCidrs := []map[string]any{}
 
 	for _, cidr := range config.GetCIDRs() {
-		formattedCidrs = append(formattedCidrs, map[string]any{"cidr": cidr})
+		if strings.Contains(cidr, "-") {
+			ipRange := strings.Split(cidr, "-")
+			formattedCidrs = append(formattedCidrs, map[string]any{"start": ipRange[0], "end": ipRange[1]})
```

Comment:
looking at this [doc](https://docs.cilium.io/en/stable/network/lb-ipam/#cidrs-ranges-and-reserved-ips) shouldn't this be `start` and `stop` not `start` and `end`

Reply:
Thanks for the catch, fixed